src/librustc_mir/interpret/intrinsics.rs

```diff
@@ -78,6 +78,13 @@ impl<'a, 'mir, 'tcx, M: Machine<'a, 'mir, 'tcx>> InterpretCx<'a, 'mir, 'tcx, M>
                 let id_val = Scalar::from_uint(type_id, dest.layout.size);
                 self.write_scalar(id_val, dest)?;
             }
+            
```

Comment:
Tidy is failing because of whitespace on this line

Reply:
Fixed it.